.pa11yci

```diff
@@ -2,8 +2,11 @@
   "defaults": {
     "userAgent": "pa11y",
     "timeout": 120000,
-    "concurrency": 2,
+    "concurrency": 10,
```

Comment:
Not sure if this does much, but it did seem to do a tiny bit?

Reply:
Yeah, I guess I know what it's meant to do, but I'm not sure what the actual effect is in practice. I seems not to run 10x faster.